templates/taxi/car_list.html

```diff
@@ -6,12 +6,19 @@ <h1>Car list</h1>
     <ul>
       {% for car in car_list %}
         <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
+          <a href="{% url "taxi:car-detail" pk=car.id %}">{{ car.id }}</a>
```

Comment:
Inconsistent use of quotes in the `{% url %}` tag. It's better to use single quotes consistently, as in `{% url 'taxi:car-detail' pk=car.id %}`.